### ISSUE TITLE
[OS X] Bungloo stops fetching new posts

### ISSUE THREAD

**Issue by mplorentz** [jeena/Bungloo#235]
Periodically Bungloo stops fetching new posts and only restarting the app gets it working again. May be triggered by computer sleeping or losing internet connection.

**Comment by dblugeon** [comment]
What is your mac os version ? On my MBP, I don't have your symptoms.

**Comment by mplorentz** [comment]
I'm on 10.8.4

**Comment by dblugeon** [comment]
do you have a lock session and lock keychain ?

**Comment by mplorentz** [comment]
I'm not sure what those are, I haven't done anything special with my keychain though, should have to default settings.

**Comment by dblugeon** [comment]
Ok can you try this settings ? 
- open keychain access
- right-clic on session
- update keychain setting "session"
- check these two checkboxes.
- launch the sleep
- awake the mac
- your mac tell about your password for bungloo (and other apps)
- check the behavior of bungloo update.

**Comment by mplorentz** [comment]
I don't see anything labeled "session" to right-click on. Is it on the main Keychain Access window?

**Comment by jeena** [comment]
It happens fo me on Linux too. I'm not sure yet how to fix it.

**Comment by dblugeon** [comment]
@mplorentz yes, it is on main keychain access window. On the top right (there is too a system session keychain)

@jeena Maybe, there is same symptom with different causes. For linux, Bungloo is in python+QT, on Mac is Objective-c+webview.
